cluster/images/hyperkube/cni-conf/10-containernet.conf

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "containernet",
+  "type": "flannel",
+  "delegate": {
+        "bridge": "cni0",
```

Comment:
If I remember correctly the name cbr0 bridge is used now by kubenet plugin. In my understanding there are expectations attached to cbr0 bridge. I thought it it is better to keep separate to avoid confusion, e.g. there are command line options that mention the name CBR0.